boefjes/boefjes/plugins/kat_dns/main.py

```diff
@@ -17,44 +18,50 @@ class ZoneNotFoundException(Exception):
     pass
 
 
+def get_record_types() -> List[str]:
+    dns_record_types = ["A", "AAAA", "CAA", "CERT", "RP", "SRV", "TXT", "MX", "NS", "CNAME", "DNAME"]
+    requested_record_types = getenv("RECORD_TYPES", False)
```

Comment:
It's better to return an empty string instead of a boolean, this would make it type-safe (and thus more clear IMHO)

```suggestion
    requested_record_types = getenv("RECORD_TYPES", "")
```